support-frontend/app/controllers/SiteMap.scala

```diff
@@ -24,13 +24,13 @@ class SiteMap(
   private def supportLandingPages()(implicit req: RequestHeader) = {
     <url>
       <loc>{
-        routes.Application.showcase().absoluteURL(secure = true)
+        routes.Application.showcase("uk").absoluteURL(secure = true)
```

Comment:
Could use `Country.UK.alpha2` instead of `"uk"` here, up to you.

Reply:
I don't feel confident enough with the Scala code to make a call on this - so I have left it as it is.